frontend/app/src/pages/settings/api-keys/exchanges/index.vue

```diff
@@ -155,27 +161,26 @@ onMounted(async () => {
   }
 });
 
-const headers: DataTableHeader[] = [
+const headers: DataTableColumn[] = [
```

Comment:
already made comment about this in the other PR...

Reply:
Yes, since this is based on the other PR, it makes sense to only review the commits that are not on the previous one :)